### DISCUSSION TITLE
High contrast in dark mode

### DISCUSSION THREAD

**Discussion by picklebento** [community/community#3134] [Other Feature Feedback, Questions, & Ideas] [answered]
My colleagues and I all feel that the GIthub dark mode has higher contrast than other dark modes. My eyes still hurt a bit when using Github at night. This might be subjective and biased, but I do want to provide the screenshots. To be MacOS and IntelliJ feel the most comfortable, Slack feels decently good, but Github feels intensive and stressful.

# Github 
![WX20210107-204004](https://user-images.githubusercontent.com/1175468/103975574-4ea6df00-5129-11eb-9721-15c8bcf2a5ef.png)
# Slack
![WX20210107-204016](https://user-images.githubusercontent.com/1175468/103975577-4f3f7580-5129-11eb-8996-78826bac7e6e.png)
# IntelliJ
![WX20210107-204625](https://user-images.githubusercontent.com/1175468/103975632-6f6f3480-5129-11eb-8a67-53b6d283e73b.png)
# MacOS/Finder
![WX20210107-204814](https://user-images.githubusercontent.com/1175468/103975714-afceb280-5129-11eb-925c-6567dd1ea6f8.png)

**Comment by evi-liu** [comment, marked as answer]
This space is currently reserved for feedback on Discussions and GitHub for Mobile. This is noted in our [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).
